composer.json

```diff
@@ -16,18 +16,18 @@
     "vlucas/phpdotenv": "^v5.3.0"
   },
   "require-dev": {
-    "blumilksoftware/codestyle": "^0.4.1"
+    "blumilksoftware/codestyle": "^0.4.3"
   },
   "scripts": {
     "build": [
       "rm -rf ./public/api",
       "sh -c 'cd backend && php index.php'"
     ],
     "populate": [
-      "sh -c 'cd backend && php index.php -r'"
+      "sh -c 'cd backend && php index.php --populate'"
     ],
     "fetch": [
-      "sh -c 'cd backend && php index.php -f'"
+      "sh -c 'cd backend && php index.php --fetch'"
```

Comment:
Also it is good moment to describe how these commands work. E.g. in the readme file.

Reply:
Maybe let's create custom command like `appinfo` that will print out help for them, and mention existence of this helper in readme?

Reply:
Actually I'll consider that in separate issue.

I've started writing some code to that and I realized I might create few more classes for handling names, description, extended descriptions. They might also handle behaviors themselves.

It might be useful if we want to limit application functions per environment. We might not want to have `populate` command on deploy, because it requires human action. It will also reduce `if{}elseif{}[...].`.